protocols/grpc/agent.proto

```diff
@@ -31,6 +31,8 @@ service AgentService {
 	rpc ListProcesses(ListProcessesRequest) returns (ListProcessesResponse);
 	rpc UpdateContainer(UpdateContainerRequest) returns (google.protobuf.Empty);
 	rpc StatsContainer(StatsContainerRequest) returns (StatsContainerResponse);
+	rpc PauseContainer(PauseContainerRequest) returns (google.protobuf.Empty);
```

Comment:
Should these 2 be moved down to the "misc" section I wonder?

Reply:
I'm not sure, update and stats are also commands, so I placed pause and resume here